website/_data/compatibility.yml

```diff
@@ -0,0 +1,82 @@
+# An overview of Error Prone Support releases, along with compatible Error
+# Prone releases. This data was generated by `generate-version-compatibility-overview.sh`.
```

Comment:
This script takes quite some time to run (and that'll go up due to it's quadratic nature :speak_no_evil:), so committing the result is the only realistic option.